x-pack/test/upgrade/config.ts

```diff
@@ -27,7 +27,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       require.resolve('./apps/graph'),
       require.resolve('./apps/logs'),
       require.resolve('./apps/maps'),
-      require.resolve('./apps/reporting'),
```

Comment:
Head's up @elastic/kibana-qa: Reporting will no longer use snapshot telemetry, hence the removal of these tests which depended on services from that.

Reply:
Update, will rethink this. It probably makes sense to keep the `enabled/available` logic in the legacy Reporting telemetry.